scripts/common/common.sh

```diff
@@ -258,7 +271,11 @@ function get_shared() {
         rm common.tar.gz
     fi
     if [ -f shared/kurl-util.tar ]; then
-        docker load < shared/kurl-util.tar
+        if [ -n "$DOCKER_VERSION" ]; then
+            docker load < shared/kurl-util.tar
+        else
+            ctr -n=k8s.io images import shared/kurl-util.tar
```

Comment:
When is `-n=k8s.io` required?

Reply:
I don't think it is actually, now reading through some documentation